frame/composable-traits/src/clearing_house.rs

```diff
@@ -0,0 +1,23 @@
+//! # Clearing House
+//!
+//! Common traits for clearing house implementations
+use frame_support::pallet_prelude::DispatchError;
+
+pub trait MarginTrading {
+	/// The trader's account identifier type
+	type AccountId;
+	/// The asset identifier type
+	type AssetId;
+	/// The balance type for an account
+	type Balance;
+
+	/// Add margin to a user's account
+	///
+	/// Assumes margin account is unique to each wallet address, i.e., there's only one margin
+	/// account per user.
+	fn add_margin(
+		acc: &Self::AccountId,
```

Comment:
This entirely comes down to personal opinion, but I would use the full name 'account' over 'acc'.